README.md

```diff
@@ -387,6 +388,11 @@ Allows changing of the textColor of the date picker. Has effect only when `displ
 <RNDateTimePicker textColor="red" />
 ```
 
+#### `accentColor` (`optional`, `iOs only`)
+
+Allows changing the accentColor (tintColor) of the date picker.
```

Comment:
I'm wondering, why did you choose to name this `accentColor` and not `tintColor`? Thanks

Reply:
I think `tintColor` is very `iOS` specific. datetimepicker might support setting an `accentColor` for android too, in the future.

I thought about three options:
* `tintColor` -> Good for people who are familiar with iOS programming, maybe not descriptive for the rest.
* `primaryColor` -> Good for most people, but maybe misleading without `secondaryColor` 
* `accentColor` -> I find `accentColor` very descriptive.

Please let me now if you want to change the API.